concepts/booleans/about.md

```diff
@@ -19,18 +75,43 @@ values into a boolean value.
 
 Unlike some other languages, `0` is not `false` and non-zero is not `true`.
 
-Rust supports three [logical binary operators][logical binary operators]: `&` (Logical And), `|` (Logical Or), `^` (Logical Xor).
+Rust supports three [logical binary operators][logical binary operators]: `&`
```

Comment:
"binary operator" usually means "an operator that takes two operators". (as opposed to unary and ternary operators).

I suggest "bitwise operator" as a less ambiguous term here.